src/rcx/src/extmeasure.cpp

```diff
@@ -249,23 +249,19 @@ ext2dBox* extMeasure::addNew2dBox(dbNet* net,
 {
   ext2dBox* bb = _2dBoxPool->alloc();
 
+  std::array<int, 2> bb_ll;
+  std::array<int, 2> bb_ur;
```

Comment:
Now that you've refactored this I'm fairly certain that bb_ll and bb_ur stand for (bounding box lower left) and (bounding box upper right). Which means that these variable are likely storing rects. 

ODB has a class for storing these call odb::dbRect that we could potentially move these to. However, we could do that in another PR.

As another note, chip people's graphics origin is the lower left corner i.e. (0,0 is the lower left pixel on the screen) as opposed to most games and other graphics programming where 0,0 is the top left corner.

Reply:
Mathematical convention is much older than either and aligns with EDA usage.

Reply:
Not arguing with the convention, but I find most non-EDA computer people are familiar with the upper left convention, and can cause confusion if not clarified.

Reply:
agreed

Reply:
Resolving, appreciate the clarification, could be a good addition to the developer intro docs.